lib/Doctrine/Common/DataFixtures/Executor/PHPCRExecutor.php

```diff
@@ -14,26 +14,26 @@
 class PHPCRExecutor extends AbstractExecutor
 {
     /**
-     * Construct new fixtures loader instance.
-     *
-     * @param DocumentManager $dm DocumentManager instance used for persistence.
+     * @var DocumentManagerInterface
      */
-    public function __construct(DocumentManager $dm, PHPCRPurger $purger = null)
+    private $dm;
+
+    /**
+     * @param DocumentManagerInterface $dm     manager instance used for persisting the fixtures
+     * @param PHPCRPurger              $purger to remove the current data if append is false
+     */
+    public function __construct(DocumentManagerInterface $dm, PHPCRPurger $purger = null)
     {
+        parent::__construct($dm);
+
         $this->dm = $dm;
         if ($purger !== null) {
-            $this->purger = $purger;
-            $this->purger->setDocumentManager($dm);
+            $purger->setDocumentManager($dm);
+            $this->setPurger($purger);
         }
-        parent::__construct($dm);
     }
 
-    /**
-     * Retrieve the DocumentManager instance this executor instance is using.
-     *
-     * @return \Doctrine\ODM\PHPCR\DocumentManager
-     */
-    public function getObjectManager()
+    public function getObjectManager(): DocumentManagerInterface
```

Comment:
This is a a BC break: we still have `1.3.x` as branch-alias for master, so we shouldn't break BC (yet).